Description: Additional optional dependency from Acegi Security which is required 
 Jenkins - this is due to use of later versions of Spring in the Acegi build on 
 Ubuntu/Debian
Author: james.page@canonical.com
Forwarded: not-needed

Index: jenkins/core/pom.xml
===================================================================
--- jenkins.orig/core/pom.xml	2011-11-22 08:20:35.783223924 +0000
+++ jenkins/core/pom.xml	2011-11-22 08:20:36.059223700 +0000
@@ -684,6 +684,12 @@
       <version>2.5</version>
     </dependency>
     <dependency>
+      <!-- Optional dependency for Acegi Security that Jenkins needs -->
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-webmvc</artifactId>
+      <version>2.5</version>
+    </dependency>
+    <dependency>
       <groupId>org.springframework</groupId>
       <artifactId>spring-core</artifactId>
       <version>2.5</version>
